### DISCUSSION TITLE
2in13 250x122 not in construktor list

### DISCUSSION THREAD

**Discussion by kpc397** [olikraus/u8g2#2153] [General]
Hello
I try to use the examples in U8g2 -but the compilation failed because there is no valid construktor named.
What can I do to use my Waveshare 2.13 inch 250x122 HAT?
Thanks

**Comment by olikraus** [comment]
E-Paper support in u8g2 has been stopped as there are better libraries for e-paper displays. 
Libraries from user @ZinggJM might include support for this display.
